apps/explorer/lib/explorer/etherscan.ex

```diff
@@ -115,7 +118,7 @@ defmodule Explorer.Etherscan do
     |> Chain.where_transaction_has_multiple_internal_transactions()
     |> InternalTransaction.where_is_different_from_parent_transaction()
     |> InternalTransaction.where_nonpending_block()
-    |> Repo.all()
+    |> Repo.replica().all()
```

Comment:
If we don't set env variables that define a different db connection for the replica, it would fall back to the default one, so if we do nothing, it would keep working as intended.